fuelup-init.sh

```diff
@@ -76,6 +78,36 @@ main() {
         esac
     done
 
+    if [ "$prompt_modify" = "yes" ]; then
+        case $SHELL in
+            */bash)
+                SHELL_PROFILE=$HOME/.bashrc
+                ;;
+            */zsh)
+                SHELL_PROFILE=$HOME/.zshrc
+                ;;
+            */fish)
+                SHELL_PROFILE=$HOME/.config/fish/config.fish
+                ;;
+            *)
+                warn "Failed to detect shell; please add ${FUELUP_DIR}/bin to your PATH manually."
```

Comment:
If this is hit, then subsequent logic to prompt the user should be skipped, and `$allow_modify` set to `no`.